types/params.go

```diff
@@ -85,6 +86,16 @@ type SynchronyParams struct {
 	MessageDelay time.Duration `json:"message_delay,string"`
 }
 
+// TimeoutParams configure the timings of the steps of the Tendermint consensus algorithm.
+type TimeoutParams struct {
+	Propose                   time.Duration `json:"propose,string"`
+	ProposeDelta              time.Duration `json:"propose_delta,string"`
+	Vote                      time.Duration `json:"vote,string"`
+	VoteDelta                 time.Duration `json:"vote_delta,string"`
+	Commit                    time.Duration `json:"commit,string"`
+	EnableCommitTimeoutBypass bool          `json:"enable_commit_timeout_bypass"`
```

Comment:
This still comes across as a little confusing / wordy to me which means it will most likely be even more confusing to the developers who are using this. Maybe we could simplify this down to `BypassCommitTimeout` instead of having the extra `Enable` there.